src/h5web/visualizations/shared/Toggler.tsx

```diff
@@ -18,8 +20,10 @@ function Toggler(props: Props): JSX.Element {
       aria-checked={value}
       onClick={onChange}
     >
-      {value ? <FiToggleRight /> : <FiToggleLeft />}
-      <span className={styles.togglerLabel}>{label}</span>
+      <span className={styles.box}>
```

Comment:
The `box` is the element that actually looks like a button.

Reply:
I plan on sharing the `box` code between normal buttons and togglers, eventually.